### ISSUE TITLE
Female voice not as good as male voice

### ISSUE THREAD

**Issue by gozluklufrodo** [espnet/espnet#3021]
I have trained the tacotron model with 3 different person voices. Male voice and the girl voice, which is very loud, gave way better results compared to the girl voice which is a little bit quiet. Is this a problem while training this model, or there could be problems in the audio dataset. I looked at the data but it seems cleaner than the other 2 datas.

**Comment by kan-bayashi** [comment]
You can check the following points:
- The length of the pause
- The amount of training data
- Transcription correctness

Since we perform normalization over the training data, the amplitude is not a problem if that is consistent over the training data.
If the volume is changed utterance-by-utterance, you should consider the amplitude normalization.

**Comment by FaridHaziyev** [comment]
After training for 220 epoch, it is diverged and I got NaN in the loss

**Comment by gafsd** [comment]
What do you normally use for amplitude normalisation?

**Comment by FaridHaziyev** [comment]
I have removed the silences in the audios and now I can see obvious improvement.
![telegram_16_F_2275111 ep 55](https://user-images.githubusercontent.com/15376005/109924442-b4eb5080-7cd9-11eb-9712-a01b72595ecc.png)
![telegram_16_F_2275111 ep 77](https://user-images.githubusercontent.com/15376005/109924460-bc125e80-7cd9-11eb-8476-053905943b2e.png)
